### PR TITLE
fix(cli): crux agent address env var

### PR DESCRIPTION
**Pull request by c3ppc3pp**

### CODE REVIEW

File: golang/pkg/cli/container_defaults.go [hunk 1]
```diff
@@ -53,8 +53,8 @@ func GetCrux(settings *Settings) *containerbuilder.DockerContainerBuilder {
 			fmt.Sprintf("KRATOS_ADMIN_URL=http://%s:%d",
 				settings.Containers.Kratos.Name,
 				settings.SettingsFile.KratosAdminPort),
-			fmt.Sprintf("CRUX_UI_URL=localhost:%d", settings.SettingsFile.TraefikWebPort),
-			fmt.Sprintf("CRUX_AGENT_ADDRESS=localhost:%d", settings.SettingsFile.CruxAgentGrpcPort),
+			fmt.Sprintf("CRUX_UI_URL=%s:%d", "localhost", settings.SettingsFile.TraefikWebPort),
```
Comment (m8vago) [inline golang/pkg/cli/container_defaults.go:56]: it looks less readable for me

**Review by m8vago — APPROVED** [review]

File: golang/pkg/cli/container_defaults.go (hunk 1 above)
Comment (c3ppc3pp) [reply golang/pkg/cli/container_defaults.go]: fixed, thanks

File: golang/pkg/cli/container_defaults.go [hunk 2]
```diff
@@ -374,7 +376,7 @@ func GetKratosPostgres(settings *Settings) *containerbuilder.DockerContainerBuil
 func GetBasePostgres(settings *Settings) *containerbuilder.DockerContainerBuilder {
 	basePostgres := containerbuilder.
 		NewDockerBuilder(context.Background()).
-		WithLogWriter(nil).
+		//		WithLogWriter(nil).
```
Comment (robot9706) [inline golang/pkg/cli/container_defaults.go:379]: Comment line :o

**Review by robot9706 — CHANGES_REQUESTED** [review]

File: golang/pkg/cli/container_defaults.go (hunk 2 above)
Comment (c3ppc3pp) [reply golang/pkg/cli/container_defaults.go]: Nice catch! Reverted.

**Review by robot9706 — APPROVED** [review]
LGTM 🦥

**Pull request merged**